guides/Install-API-instance.md

```diff
@@ -0,0 +1,109 @@
+# Install an instance of OpenFisca Web API automatically
+
+This repository provides a way to install the OpenFisca Web API automatically on a machine that exposes a SSH connection.
+
+This installation method uses [Ansible](https://www.ansible.com/), a well-known open-source tool enabling infrastructure as code, and is implemented as the [`openfisca-api` _role_](../ansible/roles/openfisca-api/) which is called by the [`openfisca-api` _playbook_](../ansible/openfisca-api.yml).
+
+For now, this _playbook_ installs an instance of OpenFisca Web API parametered to run [OpenFisca-France](https://github.com/openfisca/openfisca-france). More configuration parameters will be added in the near future, to allow installing one or many other country packages.
+
+## Requirements
+
+The Ansible tool must be installed on a machine named [control node](https://docs.ansible.com/ansible/latest/network/getting_started/basic_concepts.html#control-node) in Ansible parlance that will install OpenFisca Web API on a [managed node](https://docs.ansible.com/ansible/latest/network/getting_started/basic_concepts.html#managed-nodes). The control node can be, for example, your development machine.
+
+To install Ansible, check [its documentation](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html).
+
+To check that Ansible is well installed, try to run the following command:
+
+```bash
+ansible --version
+
+# It should output something like:
+
+# ansible [core 2.11.2]
+#   config file = /etc/ansible/ansible.cfg
+#   [...]
+```
+
+For information, during the redaction of this guide, we used Ansible version 2.11.2 with Python 3.9.4 (Ansible is written in Python). Note that the Python version used to run Ansible is not related to the [Python version recommended by OpenFisca Core](https://github.com/openfisca/openfisca-core#environment) (which is Python 3.7 at the time we write this guide).
+
+The Ansible _playbook_ is designed to be run on a managed node that runs [Ubuntu 20.04.2.0 LTS (Focal Fossa)](https://releases.ubuntu.com/20.04/), the latest long term support (LTS) version at the time the _playbook_ was written.
+
+## Configuration
+
+The default values of the variables used by the _playbook_ are defined in [`defaults/main.yml`](../ansible/roles/openfisca-api/default/main.yml).
```

Comment:
From an end user point of view, it would seem more appropriate to me to explain how to change the values through an inventory instead of explaining where the _defaults_ values are stored 🙂